libcontainer/cgroups/v1_utils.go

```diff
@@ -0,0 +1,304 @@
+package cgroups
+
+import (
+	"bufio"
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+	"strings"
+)
+
+// Functions in this source file are specific to cgroupv1,
```

Comment:
Can we split pkg

Reply:
let me try

Reply:
I was able to do that. The problem, though, is all the existing users of runc need to adopt. I'm not sure how disruptive it is.

Reply:
oops, forgot the tests :)